### ISSUE TITLE
aria-labelledby/label not being announced when tabbing through.

### ISSUE THREAD

**Issue by joe-allen-89** [adaptlearning/adapt-contrib-matching#188]
### Subject of the issue/enhancement/features
When you tab through a page of questions, the screen reader will announce the header/ariaQuestion for mcq/gmcq but doesn't with the matching component.

### Your environment
* FW v5.40.3
* Latest Firefox
* Windows, NVDA

### Steps to reproduce
Create a course and use the screen reader to tab through the page.

### Expected behaviour
As with MCQ/GMCQ the header should be read out

### Actual behaviour
The header is not included and only the prefix/text input is announced
